.github/workflows/susemanager-sls-compatible-python26-syntax.yml

```diff
@@ -0,0 +1,40 @@
+name: Check for non-compatible Python 2.6 in susemanager-sls Salt modules
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths: 'susemanager-utils/susemanager-sls/src/**/*'
+
+jobs:
+  susemanager_sls_check_non_python26_syntax:
+    runs-on: ubuntu-latest
+    container: centos:6
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v1
+
+      - name: Configure repos from CentOS vault
+        run: |
+          cd /etc/yum.repos.d/
+          sed -i 's/mirror.centos.org\/centos\/$releasever/vault.centos.org\/6.10/g' CentOS-Base.repo
+          sed -i 's/#baseurl/baseurl/g' CentOS-Base.repo
+          sed -i '/mirrorlist.*/d' CentOS-Base.repo
+
+      - name: Configure Salt products:old repos
+        run: |
+          yum install -y git wget
+          cd /etc/yum.repos.d/
+          wget http://download.opensuse.org/repositories/systemsmanagement:/saltstack:/products:/old/RHEL_6/systemsmanagement:saltstack:products:old.repo
+          sed -i 's/gpgcheck=1/gpgcheck=0/g' systemsmanagement:saltstack:products:old.repo
+
+      - name: Install old Salt from products:old
+        run: |
+          yum install -y salt
+
+      - name: Execute tests to check Python 2.6 compatiblity of custom Salt modules
+        run: |
+          cd $GITHUB_WORKSPACE/susemanager-utils/susemanager-sls/src/
+          # Exclude tests and uyuni_config.py modules (which are for server and not included in susemanager-sls)
+          TESTS_LIST=$(find .|grep py$ | grep -v "^./tests" | grep -v "uyuni_config.py")
```

Comment:
In the future we can change this to only run on changed files. I guess that's what we'll do when we add a linter etc.

Reply:
Yes, fully agree.